src/main/webapp/WEB-INF/jsp/fragments/header.jsp

```diff
@@ -0,0 +1,26 @@
+
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%@ taglib prefix="auth" uri="/WEB-INF/auth.tld" %>
+
+<nav class="navbar navbar-expand-sm bg-dark navbar-dark">
+
+    <a class="navbar-brand" href="#">
+        <img src="" alt="logo" style="width:40px;">
+    </a>
+</nav>
+<nav>
+   <auth:authorize>
```

Comment:
That was a just a test of the tag, you should hide the login form using it according to your task